common/lib/xmodule/xmodule/open_ended_grading_classes/openendedchild.py

```diff
@@ -1,21 +1,30 @@
+"""
+ORA1. Deprecated.
```

Comment:
I must not have realized I was touching ORA1 code in my original PR, so I think adding this docstring is a good idea to keep others from tripping over the code. I'd just leave your changes here in place.